bbs/main/index.php

```diff
@@ -1,5 +1,6 @@
 <?php
-	include("../lib/mainfunc.php");
+    //$cl = microtime(true);

```

Comment:
It's not a good idea to add these comments in this PR. Please REMOVE it before it is ready for actual production use.